docs/menu.xml

```diff
@@ -61,6 +61,12 @@
 </menu>
 
 <menu id="some-custom-menu">
+  <!--
+    Creates a separator w/title that is used as a header
```

Comment:
I prefer not use use abbreviations like `w/` as it might not be clear to everyone.

Also, I suggest simplifying "Creates a separator w/title that is used as a header" to simply "Create menu title."